### PR TITLE
Update multi-language-checkout.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2333](https://jira.bigcommerce.com/browse/DEVDOCS-2333)

## What changed?
Removed erroneous text

### CODE REVIEW

**Review by uwikaiddi — APPROVED** [review]

**Review by uwikaiddi — CHANGES_REQUESTED** [review]
@traci707, there are multiple instances of erroneous text still remaining in this document (ex. `<<<<<<HEAD` and `master >>>` and `=======`). All these need to be removed from the entire document.

**Comment by traci707** [comment]
Thank you Uwana for finding those other instances, I should have reviewed everything.  They have all been removed.

File: docs/stencil-docs/localization/multi-language-checkout.md [hunk 1]
```diff
@@ -154,7 +148,7 @@ BigCommerce's Optimized One-Page Checkout will currently translate displayed **C
 
 * Stencil's multi-language capabilities are limited to the strings that you specify within the theme. The Stencil framework does not currently translate content rendered from a store's database; for example, products' names.
 
->>>>>>> master
+master
```
Comment (uwikaiddi) [inline docs/stencil-docs/localization/multi-language-checkout.md:151]:
```suggestion
```

File: docs/stencil-docs/localization/multi-language-checkout.md [hunk 2]
```diff
@@ -117,7 +113,7 @@ BigCommerce's Optimized One-Page Checkout will currently translate displayed **C
 
 2. Inside your `lang` subdirectory, provide an `xx.json` file with the appropriate two-letter prefix for one or more of the supported languages. For a complete list of supported languages, see the table below. 
 
->>>>>>> master
+master
```
Comment (uwikaiddi) [inline docs/stencil-docs/localization/multi-language-checkout.md:116]:
```suggestion
```

File: docs/stencil-docs/localization/multi-language-checkout.md [hunk 3]
```diff
@@ -135,15 +130,13 @@ BigCommerce's Optimized One-Page Checkout will currently translate displayed **C
 | Chinese Traditional | `zh.json` |
 
 3. We recommend that you populate your JSON file with the checkout-specific keys-values covered in [Adding your own translation values](#adding-your-own-translation-values). It is not mandatory to translate the drop-down lists, but it is necessary to provide a consistent translation of the surrounding **Address** sections. You do not need to provide any keys-values for the **Country/State** names, whose translations already exist within the BigCommerce platform.
-<<<<<<< HEAD
+
```
Comment (uwikaiddi) [inline docs/stencil-docs/localization/multi-language-checkout.md:133]:
```suggestion
```

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**